repos/rust-lang/gha-self-hosted.toml

```diff
@@ -4,3 +4,7 @@ description = "GitHub Actions self-hosted runners infrastructure"
 bots = []
 
 [access.teams]
+infra = "write"
```

Comment:
Not needed right now for experiments ofc, but if we do end up using self-hosted runners, we should eventually probably avoid giving the whole of `t-infra` write access. To avoid the previous situation where any t-infra member could essentially arbitrarily affect CI through `rust-lang-ci/rust` :laughing: